conf.py

```diff
@@ -44,10 +38,35 @@
         'sphinx.ext.pngmath',
         'sphinx.ext.intersphinx',
         'sphinx.ext.extlinks',
+        'sphinx_gallery.gen_gallery',
 ]
 
 doctest_test_doctest_blocks = 'true'
 
+sphinx_gallery_conf = {
+    'examples_dirs': ['intro/summary-exercises/examples',
+                      'intro/matplotlib/examples',
+                      'advanced/image_processing/examples',
+                      'advanced/mathematical_optimization/examples',
+                      'packages/scikit-image/examples',
+                      'packages/statistics/examples'],
+    'gallery_dirs': ['intro/summary-exercises/auto_examples',
+                     'intro/matplotlib/auto_examples',
+                     'advanced/image_processing/auto_examples',
+                     'advanced/mathematical_optimization/auto_examples',
+                     'packages/scikit-image/auto_examples',
+                     'packages/statistics/auto_examples'],
+    'doc_module': 'scipy-lecture-notes',
```

Comment:
Do you refer to `doc_module`? I suppose that `examples_dirs`and gallery_dirs` are needed also in the Scipy lecture notes, or am I misunderstanding something?